docs/cli.rst

```diff
@@ -56,6 +56,18 @@ If you are constantly working with a virtualenv you can also put the
 bottom of the file.  That way every time you activate your virtualenv you
 automatically also activate the correct application name.
 
+Edit the activate script for the shell you use, for example:
+
+Unix Bash: ``venv/bin/activate``::
+
+    FLASK_APP=foo.py
```

Comment:
I'd rather the examples encourage the use of installed package names rather than file names. `foo` instead of `foo.py`. @untitaker thoughts?